src/main/java/uk/gov/hmcts/reform/feature/config/AuthenticationProviderConfig.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.feature.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.jdbc.JdbcDaoImpl;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import javax.sql.DataSource;
+
+@Configuration
+@AutoConfigureBefore({SecurityConfiguration.class})
+public class AuthenticationProviderConfig {
+
+    @Autowired
+    private DataSource dataSource;
+
+    @Bean(name = "userDetailsService")
```

Comment:
is this name needed?

Reply:
no not required will remove it in next PR